rpm/st2chatops.spec

```diff
@@ -8,8 +8,7 @@
 Name:           st2chatops
 Version:        %{version}
 Release:        %{release}
-Requires:       nodejs >= 6.0
-Conflicts:      nodejs < 6.0, nodejs >= 2:7.0
+Requires:       nodejs >= 8.0, nodejs < 11.0
```

Comment:
Adapting from the [documentation on `Conflicts`](https://fedoraproject.org/wiki/Packaging:Conflicts#Optional_Functionality):

> For example, if st2chatops needs nodejs to run, but will not work with nodejs that is older than 8.0:
> 
> WRONG: Conflicts: nodejs < 8.0
> RIGHT: Requires: nodejs >= 8.0

I really don't think our usecase here is what is intended for the `Conflicts` field. The `Requires` field captures exactly our intent, and it's exceptionally rare that users would try to install two different versions of node on the same system.

Reply:
> and it's exceptionally rare that users would try to install two different versions of node on the same system.

... but in reality it happens as users try to install StackStorm in a system with other software present and we want to prevent this possible conflict before it even appears.
And so having the technical possibility to limit this I see no reason why we shouldn't reject several nodejs installations to be installed in the system.

Especially when we already had reports for this specific case: https://github.com/StackStorm/st2chatops/issues/107

Reply:
Just a reminder to test if `Requires` will be enough to fail the package install if unsupported version of nodejs already present in the system.

Per previous discussion:
https://stackstorm.slack.com/archives/C029K4CBU/p1540509086000200
https://stackstorm.slack.com/archives/C029K4CBU/p1540509433000100

Reply:
[Resolved](https://github.com/StackStorm/st2chatops/pull/116#issuecomment-436475885)